doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -0,0 +1,90 @@
+.. _thread_ot_memory:
+
+OpenThread memory requirements
+##############################
+
+This page provides information about the layout and the amount of Flash and RAM memory required by the sample using the OpenThread stack.
+These values are useful to see whether your application has enough space for running on paricular platforms in different OpenThread configurations.
+
+
+Flash and RAM requirements for nRF52840 and nRF52833
+****************************************************
+
+The following tables list flash memory and RAM requirements for the provided samples built using the OpenThread libraries.
+The values change depending on the application and hardware platform.
+
+The values in the tables are provided for the CLI example, which works with all OpenThread calls.
+For this reason, these values are the best approximation of the memory requirements.
+
+Samples were compiled with the default :file: `prj.conf`, MTD variant of the CoAP Client has been compiled with the :file: `overlay-mtd.conf`.
+Below options were used to enable the multiprotocol in chosen configurations:
+
+:option:`CONFIG_MPSL=y`
+:option:`CONFIG_BT_LL_SOFTDEVICE_DEFAULT=y`
+:option:`CONFIG_BT=y`
+:option:`CONFIG_BT_SMP=y`
+:option:`CONFIG_BT_PERIPHERAL=y`
+:option:`CONFIG_BT_DEVICE_NAME="NUS_CoAP_client"`
+:option:`CONFIG_BT_DEVICE_APPEARANCE=833`
+:option:`CONFIG_BT_MAX_CONN=1`
+:option:`CONFIG_BT_MAX_PAIRED=1`
+:option:`CONFIG_BT_GATT_NUS=y`
+:option:`CONFIG_BT_SETTINGS=y`
+
+
+nRF52840 RAM and Flash memory requirements
```

Comment:
Board names should match what is in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html#board-names

So either `nRF52840 DK` or `nrf52840dk_nrf52840`.
Same goes for 833 below.
```suggestion
nRF52840 DK RAM and flash requirements
```